### PR TITLE
updated the codebase to the latest package versions

### PR DESCRIPTION
**Pull request by michaelessiet**
Hi Takuya,

I updated the package versions to the latest versions and solved an error in react-native-checkbox-reanimated. I'll fork and make a PR with the fix to the issue in that repo as well.

### CODE REVIEW

**Comment by craftzdog** [comment]
Thanks. Have you tested it on your device?

**Comment by michaelessiet** [comment]
> Thanks. Have you tested it on your device? I don't have time to test it myself at the moment

Yes, I have it works perfectly. By the way, the react-native-checkbox-reanimated needs to be updated to the latest version as well

File: package.json [hunk 1]
```diff
@@ -27,21 +27,23 @@
...
-    "react-native-screens": "^3.9.0",
-    "react-native-svg": "^12.1.1",
-    "react-native-web": "0.17.1",
+    "react-native-gesture-handler": "~2.5.0",
+    "react-native-reanimated": "~2.9.1",
+    "react-native-safe-area-context": "^4.3.1",
+    "react-native-screens": "~3.15.0",
+    "react-native-svg": "^12.3.0",
+    "react-native-web": "^0.18.8",
+    "react-stately": "^3.17.0",
```
Comment (craftzdog) [inline package.json:46]: can you tell me why `react-stately` is added?

Comment (michaelessiet) [reply package.json:46]: Oh, I ran into an error while bundling the app. It said that one of the packages had react-stately as a dependency. The app wouldn't bundle without it. So I installed it to fix the error.

**Comment by craftzdog** [comment]
Ok, thanks! Would you update the checkbox component as well?

**Comment by michaelessiet** [comment]
> Ok, thanks! Would you update the checkbox component as well?

Yes, I will check and test it out on my end and let you know if there's anything that needs to be updated

**Pull request merged**